### ISSUE TITLE
Codemirror questions

### ISSUE THREAD

**Issue by gmclelland** [rolandtoth/AdminOnSteroids#26]
Just curious, when trying the CodeMirror plugin, does lineWrapping work for you?
![edit page about pw3 dev 2016-12-09 11-13-47](https://cloud.githubusercontent.com/assets/40570/21058259/c0471332-be02-11e6-9d02-719dde7a2f91.jpg)

Also do you notice how the closing paragraph tags wrap to another line.  It doesn't show that way when using the Ckeditor view source.

Is there any way to make the source view dialog bigger/resizable?  Maybe use 90% instead of 800px?  Looks like it is defined in AdminOnSteroids/CKE/plugins/codemirror/plugin.js:67

**Comment by rolandtoth** [comment]
No, lineWrapping fails here too, I couldn't get it to work. Perhaps there's some CSS conflict here too. I had a fix earlier but others reported issues with it so I removed.

No idea about the dialog size, if you find out something about it please report :)

**Comment by gmclelland** [comment]
Thanks for the reply.

If interested, here is a demo of the line wrapping I was hoping to achieve http://codemirror.net/demo/indentwrap.html
Maybe it needs the additional code that is used in the demo?

The dialog resizing is present when you click view source button without the CodeMirror plugin.

**Comment by lesaff** [comment]
I solved this by
- disabling codemirror in AoS
- installed codemirror as a standalone CKEditor plugin

**Comment by rolandtoth** [comment]
Thanks for the info. I've made some changes and line-wrapping seems to work fine here.

Btw, does the TAB key loose editor focus there too after first usage?
